### PR TITLE
adds assignsudo problem (#9)

### PR DESCRIPTION
**Pull request by Priyankasaggu11929**
@kushaldas please review.

### CODE REVIEW

File: lymproblems/assignsudo.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+def setup():
+    "Setup problemname"
+    pass  # Nothing to do.
+
+
+def verify():
+    "Verify problemname"
+    user = "lym"
+
+    if user not in os.popen("getent group sudo").read():
```
Comment (kushaldas) [inline lymproblems/assignsudo.py]: Please use the `system` function than using os.popen directly.

Also, the solution is wrong. We don't add people to `sudo` to get sudo access.

**Review by kushaldas — CHANGES_REQUESTED** [review]

File: lymproblems/assignsudo.py (hunk 1 above)
Comment (Priyankasaggu11929) [reply lymproblems/assignsudo.py]: I will re-work on this. Thank you for the reviews/suggestion.

Comment (Priyankasaggu11929) [reply lymproblems/assignsudo.py]: @kushaldas, please review.

File: lymproblems/assignsudo.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+def setup():
+    "Setup problemname"
+    pass  # Nothing to do.
+
+
+def verify():
+    "Verify problemname"
+    user = "lym"
+
+    if system("sudo -l -U {}".format(user))[2]!=0:
```
Comment (kushaldas) [inline lymproblems/assignsudo.py]: Fix the code formatting, space before `!=`

File: lymproblems/assignsudo.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Assign sudo-user/superuser privileges to a current user "lym".
```
Comment (kushaldas) [inline lymproblems/assignsudo.py]: Explain the problem in a better words please

**Review by kushaldas — CHANGES_REQUESTED** [review]

File: lymproblems/assignsudo.py (hunk 3 above)
Comment (Priyankasaggu11929) [reply lymproblems/assignsudo.py]: @kushaldas I corrected the formatting part and re-wrote the problem statement.

**Review by kushaldas — APPROVED** [review]

**Pull request merged**